src/statd/statd.c

```diff
@@ -173,8 +184,8 @@ static int ly_add_yanger_data(const struct ly_ctx *ctx, struct lyd_node **parent
 }
 
 static int sr_ifaces_cb(sr_session_ctx_t *session, uint32_t, const char *path,
-			const char *, const char *, uint32_t,
-			struct lyd_node **parent, void *priv)
+                        const char *, const char *, uint32_t,
```

Comment:
I believe this indentation was "the kernel way"?

Reply:
I just ran the proposed indentation by jocke made in #209, but i can see that it is spaces not tabs..